recipes/mender-shell/debian-master/control

```diff
@@ -0,0 +1,13 @@
+Source: mender-shell
+Maintainer: Mender Team <mender@northern.tech>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-shell
+Conflicts: mender
+Architecture: any
```

Comment:
Question: Is this really true?

Reply:
Yes. See https://www.debian.org/doc/debian-policy/ch-controlfields.html#architecture